### DISCUSSION TITLE
Remastered Music

### DISCUSSION THREAD

**Discussion by TheAtomTwister** [OpenRCT2/OpenRCT2#19270] [Feature requests] [answered]
Allister Brimble, the guy who made the original soundtracks for RCT2, made remastered versions of the soundtracks that he uploaded to his YouTube channel including an extended version of Modern Style.  You guys might enjoy hitting him up and showing him what you all have done here and see if he wants to contribute them to OpenRCT2.

Here are some links to his remastered RCT2 music: 

Modern Style Extended:
https://www.youtube.com/watch?v=PHV2pj8m6SI&t=1s

All other tracks:
https://www.youtube.com/watch?v=ykuT2RqB_LM&t=1617s

**Comment by karst** [comment, marked as answer]
Sadly the music for Chris Sawyer's games are licensed and we cannot use it due to copyright issues. If Allister Brimble would like to provide something to the project this is of course more than welcome, but it needs to be licensed under CC BY-SA 4.0.

**Reply by TheAtomTwister** [reply]
I see.  Well, if you do manage to get the appropriate licensing for its use, then hallelujah.  In any case, thank you.
